setup.py

```diff
@@ -12,7 +12,7 @@
     author='edX',
     author_email='oscm@edx.org',
     url='https://github.com/edx/edx-drf-extensions',
-    license='AGPL',
+    license='Apache 2.0',
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Environment :: Web Environment',
```

Comment:
Sorry to be persnickety, but two lines down, it should say:
```
    'License :: OSI Approved :: Apache Software License',
```

Reply:
All set!